libs/base/utilities/src/signals/signal-from.ts

```diff
@@ -11,8 +11,35 @@ export interface ConnectableSignalOptions<T, K> extends SignalOptions<T | K> {
   initialValue?: K;
 }
 
+let _resolvingSignals: number | undefined = undefined;
+let _resolvingSignalsNotifier: StateSignal<number> | undefined = undefined;
+
+function signalResolving(state = true): void {
```

Comment:
Having two similar names for functions (`signalResolving` and `resolvingSignals`) can be confusing. Suggest to rename and maybe write a small description.

Reply:
👍